src/vector/feature.rs

```diff
@@ -835,7 +834,7 @@ impl FieldValue {
 
 pub fn field_type_to_name(ty: OGRFieldType::Type) -> String {
     let rv = unsafe { gdal_sys::OGR_GetFieldTypeName(ty) };
-    _string(rv)
+    _string(rv).unwrap_or_default()
```

Comment:
`rv` is a static string here, right?

Reply:
Yes, returns a static string, never NULL.